.travis.yml

```diff
@@ -70,8 +68,13 @@ before_install:
       # as exploratory tests for when the shipped version has automatic OpenMP support.
       # These tests will then fail and at such a time a new one should be added
       # to explicitly remove OpenMP support.
-    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then export OPENMP_EXPECTED="False";
-      else export OPENMP_EXPECTED="True"; fi
+    - if [ -z $OPENMP_EXPECTED ]; then
```

Comment:
Still, why do we need this at al? Variables defined in the env section are available to `os.environ` for later scripts.

Reply:
@bsipocz - because we can't define it to be True for the default linux build and False for the default False build (builds defined in the ``env: matrix:`` section not the ``matrix: include:`` section).

Reply:
Specifically for these builds:

```
 os:
     - osx
     - linux
 
 env:
   matrix:
     - PYTHON_VERSION=3.5
     - PYTHON_VERSION=3.6 SETUPTOOLS_VERSION=dev DEBUG=True
       CONDA_DEPENDENCIES='sphinx cython numpy pytest-cov'
       EVENT_TYPE='push pull_request cron'
```

Reply:
ahh, OK, now I see.  I would have then just removed the second from here leaving only the egg build in the main matrix.

Reply:
The first isn't an egg test so both would need to be taken out and four builds added to the include matrix, so I'm not sure it would simplify things that much. I'll go ahead and merge, but feel free to open a PR if you'd like to simplify things further.

Reply:
ohh, indeed, should read more carefully (or stop multitask as pay more attention to the talk I'm sitting at...)